Documentation/operations/system_requirements.rst

```diff
@@ -95,14 +95,14 @@ Ubuntu_                    >= 20.04
           GitHub issue or by creating a pull request that updates this guide.
 
 
-Flatcar
-~~~~~~~
+Flatcar on AWS EKS in ENI mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Flatcar is known to manipulate network interfaces created and managed by
-Cilium. This is especially true in the official Flatcar image for AWS EKS, and
-causes connectivity issues and potentially prevents the Cilium agent from
-booting when Cilium is running in ENI mode. To avoid this, disable DHCP on
-these interfaces and mark them as unmanaged by adding
+Cilium. When running the official Flatcar image for AWS EKS nodes in ENI
+mode, this may cause connectivity issues and potentially prevent the Cilium
+agent from booting. To avoid this, disable DHCP on the ENI interfaces and mark
+them as unmanaged.
```

Comment:
We still need to introduce the coming configuration snippet:

```suggestion
them as unmanaged. To do so, add the following configuration options to ``/etc/systemd/network/01-no-dhcp.network``::

    [Match]
    Name=eth[1-9]*
    [Network]
    DHCP=no
    [Link]
    Unmanaged=yes

Then run:
```

Then the systemctl commands